lmfit/confidence.py

```diff
@@ -409,11 +409,12 @@ def conf_interval2d(minimizer, result, x_name, y_name, nx=10, ny=10,
     """
     params = result.params
 
-    best_chi = result.chisqr
+    best_chisqr = result.chisqr
+    redchi = result.redchi
     org = copy_vals(result.params)
 
     def chi2_compare(best, current):
```

Comment:
this doesn't use `best` anymore; do we need a function to just return `current.chisqr` (I guess the answer might be yes, but then one should probably clean up the arguments)

Reply:
@reneeotten thanks -- will investigate...

Reply:
I cleaned up the code here...